### PR TITLE
Feature create `Popover` component

### PR DESCRIPTION
**Pull request by daoanhhuy26012001**
- #50 
- [x] Make a button wrap block
- [x] Pass necessary props to button(`variant`, `size`,...)
- [x] Implement creating 2 array containing images and text (`BRANDS`, `PRICES`)
- [x] Split into 2 popovers
  - If `buttonType === image` the popover contains an image button
  - Conversely, if `buttonType === text` then the popover contains the button text
- **NOTE**:
  - The three component objects I create in the storybook will correspond to: 
image
image
image
  - See more details: [Link](https://docs.google.com/document/d/1iTrT9cdKrMCjUc6aDLS514gEk7BUFj-m2SCs2wCy2Lw/edit#heading=h.7t5pbcmau6ih) 
**RESULT**
image

### CODE REVIEW

**Comment by daoanhhuy26012001** [comment]
Popover shows two types of buttons I changed the layout to make it more eye-catching

File: ReactJS/practice-one/src/constants/constant.js [hunk 1]
```diff
@@ -11,13 +11,13 @@ const VARIANTS = {
   QUATERNARY: 'quaternary'
 };
 
-const PRICE = [
+const PRICES = [
```
Comment (phattruong-agilityio) [inline ReactJS/practice-one/src/constants/constant.js:14]: Please check with the requirement

File: ReactJS/practice-one/src/components/Common/Popover/index.jsx [hunk 2]
```diff
@@ -1,32 +1,53 @@
 import { Button } from '../Button/index';
 import { BRAND } from '../../../constants/images';
-import { VARIANTS } from '../../../constants/constant';
-
+import { VARIANTS, PRICES } from '../../../constants/constant';
 import './popover.css';
 import '../Button/button.css';
 
+/**
```
Comment (phattruong-agilityio) [inline ReactJS/practice-one/src/components/Common/Popover/index.jsx:7]: It's not a common

**Comment by phattruong-agilityio** [comment]
You are building the common component for popover UI is wrong. Please help check the bootstrap https://getbootstrap.com/docs/4.0/components/popovers/
The popover should contain the area to fill the content. not contain the content.

This is my example how Popover will be used 
```

<Popover 
  button={<Button  />} // <=== your button here. its place for your button-component, click it then your popover is pop-up + display
  isOpen={true/false} 
>
  {children} <== the popover body content goes here
</Popover>

```

File: ReactJS/practice-one/src/constants/constant.js (hunk 1 above)
Comment (daoanhhuy26012001) [reply ReactJS/practice-one/src/constants/constant.js:14]: Done

**Pull request closed without merging**